tests/docker-all-versions-image/all-released-image/Dockerfile

```diff
@@ -17,16 +17,16 @@
 # under the License.
 #
 
-FROM centos:7
+FROM openjdk:8-jdk
```

Comment:
any reason what it was changed from centos7 to openjdk?

Reply:
to avoid having to pull epel (which we need for supervisor). openjdk is debian based, as has supervisor in the base repos.